tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -155,6 +155,7 @@ def get_resource_tag_value(resource, k):
 
 
 def resource_format(resource, resource_type):
+    resource = resource.lstrip('aws.')
```

Comment:
this should be resource_type = resource_type.lstrip('aws.')

Reply:
ah yea good catch